tests/model_hub_tests/pytorch/models/hf-tiny-random-models-precommit

```diff
@@ -1,5 +1,37 @@
- facebook/opt-125m,https://huggingface.co/facebook/opt-125m
- hf-tiny-model-private/tiny-random-CodeGenForCausalLM,https://huggingface.co/hf-tiny-model-private/tiny-random-CodeGenForCausalLM
+hf-tiny-model-private/tiny-random-Data2VecTextForCausalLM,https://huggingface.co/hf-internal-testing/tiny-random-Data2VecTextForCausalLM,xfail,not working
```

Comment:
here is the filtered list https://github.com/ilya-lavrenov/openvino.genai/blob/ct-beam-search/text_generation/causal_lm/cpp/continuous_batching/python/tests/models/nightly